docs/api/CallComposite/classes/SupportLanguage.md

```diff
@@ -0,0 +1,51 @@
+**CLASS**
```

Comment:
This file is part of this PR change?

Reply:
No. This change came with merge from develop/beta.